src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction(Operation operation, String fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public void setFruit(String fruit) {
+        this.fruit = fruit;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public void setOperation(Operation operation) {
+        this.operation = operation;
+    }
+
+    public enum Operation {
```

Comment:
Consider if the 'Operation' enum should be placed within the 'FruitTransaction' class. If it's used by other classes extensively and not just for 'FruitTransaction', it might be better to have it in a separate file to follow the Single Responsibility Principle (SRP) of SOLID.